static/mobile.less

```diff
@@ -15,7 +14,7 @@ label[for="nav-trigger"] {
   cursor: pointer;
   font-weight: 700;
   .border-radius;
-  z-index: 99;
+  z-index: @z-index-mid;
```

Comment:
This one doesn’t seem necessary either because the header already has a z-index.